src/docs/product/alerts-notifications/index.mdx

```diff
@@ -8,19 +8,40 @@ description: "Sentry categorizes notifications into alerts and other types of no
 
 Sentry categorizes notifications broadly into alerts and non-alert notifications.
 
-## Alerts
+## Error Alerting
 
-[Alerts](/product/alerts-notifications/alerts/) notify you about problems in your application and can be sent to many supported integrations. You can set alerts on both errors and performance data.
```

Comment:
This is a bit confusing with the new heading because we discuss alerting on both errors and performance, but the title is errors...?